package.json

```diff
@@ -95,11 +95,10 @@
 	"dependencies": {
 		"@ui5/logger": "^1.0.1",
 		"clone": "^2.1.0",
-		"dir-glob": "2.0.0",
```

Comment:
So pinning this is not required anymore with `globby` v9, right?

Reply:
Yes, we had to pin it because it changed globby's behavior. We adapted our project now to these changes. Btw. only *now* globby behaves the same as micromatch. This means only now our Memory- and FS-Adapters behave the same